pkg/ignition/ignition.go

```diff
@@ -102,5 +126,109 @@ func Initialize(c client.Client) error {
 		return errors.Errorf("ignition missing kubelet systemd unit file")
 	}
 	log.Info("extracted", "kubelete systemd unit", RenderedWorker.KubeletUnit)
-	return nil
+	RenderedWorker.BootstrapFiles, err = setupBootstrapFiles(clusterServiceCIDR)
+	return err
+}
+
+// setupBootstrapFiles creates bootstrap kubeconfig, cloud-config, kubelet cert and config files and returns their paths
+func setupBootstrapFiles(clusterServiceCIDR string) ([]string, error) {
```

Comment:
Aren't you creating the files here?

Reply:
Yes, should I rename the function to make that more clear?